apps/astro-deno-demo/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "astro-deno-demo",
+  "type": "module",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro check && astro build",
+    "preview": "deno run --allow-net --allow-read --allow-env ./dist/server/entry.mjs",
+    "astro": "astro"
+  },
+  "dependencies": {
+    "@astrojs/check": "^0.5.10",
+    "@astrojs/deno": "^5.0.1",
```

https://github.com/denoland/deno-astro-adapter/issues/10#issuecomment-2016125546
https://github.com/denoland/deno-astro-adapter/issues/13

Not sure if you have seen either of these comments?

Given the outdated NPM registry, might want to change this to:
```suggestion
    "@astrojs/deno": "@astrojs/deno": "github:denoland/deno-astro-adapter",
```

---

> Hey @siguici Thanks for taking on this issue! I have been out sick for the past few days and am just getting caught up with this discussion.
> 
> Not sure if you have seen either of these comments?

Wishing you a speedy recovery.

I've just seen the issue, but I don't think the problem lies there, considering that Astro's integration with Deno doesn't seem to be an issue. The problem seems to be more related to the use of environment variables. Deno doesn't support process.env.
However, I'll try to explore this way to see if it resolves the problem.